pageserve/README.md

```diff
@@ -0,0 +1,43 @@
+# pageserve microservice - Large Events (Knative Proof of Portability)
+
+Serve web UI and act as an API gateway to other services.
+
+These instructions are specific to this microservice. For general instructions and other information, see [the master README.md](../README.md).
+
+### Installing
+
+How to install and run this microservice for local development.
+
+Each microservice has slightly different dependencies and thus slightly different installation procedures. These instructions are specific to the pageserve microservice, but each other subfolder's README.md has an "Installing" section with more accurate instructions.
+
+Make sure your working directory is this microservice's subfolder.
+
+```sh
+cd pageserve
+```
+
+Set up and activate a virtual environment.
+
+```sh
+python3 -m venv venv && . venv/bin/activate
+```
+
+Install the required python modules.
+
+```sh
+pip3 install -r requirements.txt
+```
+
+Set any environment variables needed by the service. For the pageserve service, you need to set the following:
```

Comment:
rather than "you need to set the following", try "it needs to know the addresses of the other service modules and login tools."

Reply:
Done https://github.com/knative-portability/large-events/pull/39/commits/5b5015aa9b426548820fc25544b4840097090658 with a little extra clarification about Google sign in